hack/benchmark/internal/operation/insert.go

```diff
@@ -0,0 +1,127 @@
+package operation
+
+import (
+	"context"
+	"io"
+	"strconv"
+	"sync"
+	"sync/atomic"
+	"testing"
+
+	"github.com/vdaas/vald/apis/grpc/v1/payload"
+	"github.com/vdaas/vald/hack/benchmark/internal/assets"
+	"github.com/vdaas/vald/internal/net/grpc/codes"
+	"github.com/vdaas/vald/internal/net/grpc/status"
+)
+
+func (o *operation) Insert(b *testing.B, ctx context.Context, ds assets.Dataset) (insertedNum int) {
+	b.ResetTimer()
+	b.Run("Insert", func(b *testing.B) {
+		req := &payload.Insert_Request{
+			Vector: &payload.Object_Vector{},
+			Config: &payload.Insert_Config{
+				SkipStrictExistCheck: false,
+			},
+		}
+		b.ResetTimer()
+
+		for i := 0; i < b.N; i++ {
+			v, err := ds.Train(i % ds.TrainSize())
+			if err != nil {
+				b.Error(err)
+				continue
+			}
+
+			req.Vector.Id, req.Vector.Vector = strconv.Itoa(i), v.([]float32)
+
+			loc, err := o.client.Insert(ctx, req)
+			if err != nil {
+				st, _ := status.FromError(err)
+				if st.Code() != codes.AlreadyExists {
+					statusError(b, int32(st.Code()), st.Message(), st.Details()...)
+				}
```

Comment:
Thinking how should we handle already exists case... 
I think the insert performance between exists and non-exists is different; depending on the dataset, the benchmark result might be different.
What do you think?

Reply:
Normally, it should also output an error in the case of Already Exists.
However, the benchmark executes the function multiple times to get the results. Therefore, Already Exists will always occur, so it is intentionally omitted in this case.

Reply:
In order to properly get the benchmark results in the case of Already Exists, it is necessary to get them in a stand-alone operation rather than in e2e.

Reply:
It is difficult... 
As you said the already exists error will always occur, so in this benchmark test, it is actually benchmarking the insert function with already exists case, which is different with what I thought 🤔

I may be wrong, but what do you think?